spec/rubocop/config_spec.rb

```diff
@@ -56,7 +56,7 @@
 
           it "uses the single digit Rails version in #{file_name}" do
             content =
-              <<-HEREDOC
+              <<~HEREDOC
```

Comment:
Should we take the chance to rename this `LOCKFILE`?

Reply:
Thanks for the great suggestion! I updated it.